gamestonk_terminal/config_terminal.py

```diff
@@ -90,3 +90,7 @@
 API_COINBASE_KEY = os.getenv("GT_API_COINBASE_KEY") or "REPLACE_ME"
 API_COINBASE_SECRET = os.getenv("GT_API_COINBASE_SECRET") or "REPLACE_ME"
 API_COINBASE_PASS_PHRASE = os.getenv("GT_API_COINBASE_PASS_PHRASE") or "REPLACE_ME"
+
```

Comment:
@jmaslek didn't you forget to add:

```
APCA_API_BASE_URL = os.getenv("GT_APCA_API_BASE_URL") or "REPLACE_ME"
APCA_API_KEY_ID = os.getenv("GT_APCA_API_KEY_ID") or "REPLACE_ME"
APCA_API_SECRET_KEY = os.getenv("GT_APCA_API_SECRET_KEY") or "REPLACE_ME"
```

in here?

Reply:
So those actually just need to live in environment.  The api doesn't take in the password and stuff.